packages/sdks/web-component/src/lib/helpers/helpers.ts

```diff
@@ -563,3 +565,10 @@ export const clearPreviousExternalInputs = () => {
     .querySelectorAll('[data-hidden-input="true"]')
     .forEach((ele) => ele.remove());
 };
+
+export const escapeMarkdown = (s: string) => {
```

Comment:
Didn't we agree that we are not manipulating the markdown string in the component?

Reply:
I don't see an alternative. Changing this in the backend would not be backward compatible for regular text components.

Also, add another change to validate `shouldHandleMarkdown`.

Reply:
This means some markdown features will no longer be supported 
I don’t think this is a good direction, as it could break other syntax we aren’t aware of

Additionally, what do we expect to happen if someone adds **aaa** in the console editor?